addons/product/product_view.xml

```diff
@@ -128,13 +113,19 @@
                                     <group groups="product.group_uos" string="Unit of Measure">
                                         <field name="uos_id"/>
                                         <field name="uos_coeff"/>
-                                        <field name="mes_type"/>
                                     </group>
                                 </group>
                                 <group name="website_and_pos" col="2">
                                 </group>
-                                <separator string="Description for Quotations"/>
-                                <field name="description_sale" placeholder="note to be displayed on quotations..."/>
+                                <group name="email_template_and_project" attrs="{'invisible':[('type', '!=', 'service')]}"/>
```

Comment:
explain a problem in code why you need this, i understand it is trick to solve a small problem of display a label,